test/components/ColumnCard.nuxt.test.ts

```diff
@@ -0,0 +1,35 @@
+import { screen } from '@testing-library/vue'
+import { renderSuspended } from '@nuxt/test-utils/runtime'
+import { ColumnCard } from '#components'
+import { taskTruncatedIdLength } from '~/constants/tasks'
+
+describe('<ColumnCard />', () => {
+  it('should render the title (link to a new tab), status, id (truncated), and labels', async () => {
+    const props = {
+      id: '786cfb329a8a63dadda30caaa38437b870ab6bf8',
+      title: 'Title',
+      labels: ['label 1', 'label 2'],
+      status: {
+        name: 'Open',
+        icon: {
+          name: 'octicon:issue-opened-16',
+          class: '',
+        },
+      },
+      href: 'https://example.com',
+    }
+    await renderSuspended(ColumnCard, {
+      props,
+    })
```

Comment:
nit: new line for separation between "chapters": setup, execution, finalization. Also, consolidate the execution part into a single line because we can.

```suggestion

    await renderSuspended(ColumnCard, { props })
```

Reply:
I consider rendering the component part of the "setup" or "arrange" chapter. This is also how it is treated in [this example](https://testing-library.com/docs/react-testing-library/example-intro#quickstart) from the `testing-library` docs.

I guess you could count it as "execution" or "act" in this case since there's no user action.

Updated as suggested.

Reply:
I think you're more right than me here. Since this is just a minor style nit I'll approve this version now, but feel free to revert it in any later patch. What matters most to me is alignment and knowing we have a systematic approach, so as far as I'm concerned that much was achieved here. :+1: